# Uses "get_pmap" service provided by "pmap_files_server" node.
string pmap_filename
string pmap_frame_id
# Uses "get_octomap" service provided by  some "octomap_store" node.
string octomap_filename
string octomap_frame_id

float64 height_mean
float64 height_var
---
octomap_msgs/Octomap octomap
